pkg/anonymization/builder.go

```diff
@@ -0,0 +1,92 @@
+package anonymization
+
+import (
+	"regexp"
+	"strings"
+
+	"github.com/openshift/api/insights/v1alpha1"
+	v1 "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+	networkv1client "github.com/openshift/client-go/network/clientset/versioned/typed/network/v1"
+	"github.com/openshift/insights-operator/pkg/config/configobserver"
+	"k8s.io/client-go/kubernetes"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+	k8snet "k8s.io/utils/net"
+)
+
+type AnonBuilder struct {
+	anon     Anonymizer
+	networks []string
+}
+
+func (b *AnonBuilder) AddSensitiveValue(value, placeholder string) *AnonBuilder {
+	if b.anon.sensitiveValues == nil {
+		b.anon.sensitiveValues = make(map[string]string)
+	}
+	v := strings.TrimSpace(value)
+	if value != "" {
+		b.anon.sensitiveValues[v] = placeholder
+	}
```

Comment:
Perhaps we could do at the beginning of the function:
```
if value == "" {
  return b
}
```

Reply:
Solved in bc66284efa66957e06efbc3efd2c1f893793085e, thanks for the review